### ISSUE TITLE
Resources.cs causes issues for updating Android support libraries

### ISSUE THREAD

**Issue by IlSocio** [aritchie/userdialogs#142]
I just upgraded to latest AppCompat v23.2.1 
https://www.nuget.org/packages/Xamarin.Android.Support.v7.AppCompat/
and I'm experiencing compilation issues on Android, which seems to be caused by Acr.UserDialogs.Android.AppCompat v4.3.2

All the compilation errors I get are like the ones below:
```
1>D:\tierra-mobileapp\GUI\Droid\Resources\Resource.designer.cs(485,124,485,147): error CS0117: 'Resource.Dimension' does not contain a definition for 'design_fab_content_size'
1>D:\tierra-mobileapp\GUI\Droid\Resources\Resource.designer.cs(373,121,373,149): error CS0117: 'Resource.Color' does not contain a definition for 'design_textinput_error_color'
1>D:\tierra-mobileapp\GUI\Droid\Resources\Resource.designer.cs(1410,106,1410,111): error CS0117: 'Resource.Styleable' does not contain a definition for 'Theme'
1>D:\tierra-mobileapp\GUI\Droid\Resources\Resource.designer.cs(1411,123,1411,145): error CS0117: 'Resource.Styleable' does not contain a definition for 'Theme_actionBarDivider'
```

And looking at my Resource.Designer.cs all the errors are related to the Acr component:
`global::Acr.UserDialogs.Resource.Dimension.design_fab_content_size = global::IT.Tierra.MobileApp.Gui.Resource.Dimension.design_fab_content_size;
`

By removing the Acr.UserDialogs.Android.AppCompat I'm able to compile again without any issue.

Maybe you should delete all your values from the Resource.Designer.cs?
https://github.com/aritchie/userdialogs/blob/master/src/Acr.UserDialogs.Android.AppCompat/Resources/Resource.Designer.cs

**Comment by aritchie** [comment]
Please refrain from +1's, nuget library update requests (this is not an issue with the library), and requesting timelines for something.  Please also make sure you have read the FAQ section on the README

**Comment by IlSocio** [comment]
Ritchie, I believe this issue is related to your library.
Currently, you need to update your library on each new version of AppCompat.

By deleting the Resource.Designer.cs from your library, you'll be able to support current and any future versions of the AppCompat out-of-the-box, without being forced to release a new version of your library...

**Comment by aritchie** [comment]
I have started removing resources.cs everywhere.  I didn't actually know this would work without breaking something.  Thanks.  Will be part of 4.4

**Comment by aritchie** [comment]
fixed as part of 4.4
